src/components/RadioInput/useRadioInput.tsx

```diff
@@ -1,34 +1,41 @@
 import React, { useState } from 'react';
 import { Styles } from './styles';
 
-function useRadioInput(
-  defaultChecked?: boolean,
-) {
+/**
+ * Custom hook for radio buttons.
+ * @param {boolean} defaultChecked If checkbox should be checked by default.
+ * @returns {any} a component, the state and function to update state.
+ */
+function useRadioInput(defaultChecked?: boolean): any {
```

Comment:
I think the return type can be inferred automatically, consider leaving return type (`any`) out entirely or using the correct return type.

Reply:
This was when I was yet to accept eslint config from the app into my vscode. The default one I had was messing me up. I've updated it.